game.js

```diff
@@ -0,0 +1,143 @@
+var sizeOfOneSquare = 40, sizeOfFoundation = 3;
+var canvas = document.getElementById("main"), context = 
+canvas.getContext("2d");
```

Comment:
why you set enter here

Reply:
it isn't me, I write it in git and it make an enter

Reply:
delete it